.eslintrc

```diff
@@ -1,8 +1,14 @@
 {
   "extends": [
     "airbnb",
-    "airbnb/hooks"
+    "airbnb/hooks",
+    "plugin:react/recommended",
+    "plugin:react/jsx-runtime",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:react-hooks/recommended"
   ],
+  "plugins": ["react", "import"],
```

Comment:
and this (already inherited from airbnb, no?)

Reply:
yep.